### ISSUE TITLE
Help with my transaction

### ISSUE THREAD

**Issue by fabiangarrett93** [keybase/client#21106]
Help with my transaction, I made a transaction, but I didn't put public memo on my transaction, now what do I do?

**Comment by munnamizan1494** [comment]
If the recipient required memo for trangection you must put it.

**Comment by fabiangarrett93** [comment]
yes, but I made my transaction without public memo, how to recover a transaction?

**Comment by munnamizan1494** [comment]
> yes, but I made my transaction without public memo, how to recover a transaction?

If the trangection was sent you can't do anything now. It means you lost your funds.

**Comment by fabiangarrett93** [comment]
are you sure about this?

**Comment by munnamizan1494** [comment]
> are you sure about this?

Yes! If the recipient required memo you must put memo. If you sent it to an exchange you definitely lost it. Sorry to say that.

**Comment by exegg** [comment]
If you sent the XLM to an exchange go ahead and contact their customer support. Usually they'll be able to fix this.
